components/admin_console/generated_setting.tsx

```diff
@@ -65,9 +62,10 @@ export default class GeneratedSetting extends React.Component {
 
         let text = this.props.value;
```

Comment:
Probably you should define here that text is an string or a React.ReactNode, and set the text as was set before.

This is not going to show the placeholder when the value is an empty string.

Reply:
ReactNode already covers the string case, so changing it to React.ReactNode is enough. Thank you for the tip! Corrected on commit 3fee326